.gitignore

```diff
@@ -5,3 +5,4 @@
 .vscode
 venv/
 dist
+ai-missions/*/skills/*/src/common
```

Comment:
why is this ignored?

Reply:
we copy commos as part of the build (for individual skills) here -> https://github.com/CognitiveScale/cortex-fabric-examples/blob/1393fb9708a227f85bdcb920a04719122c6b0100/ai-missions/high-risk-flu-shot-mission/skills/send_virtual_care_notification/Makefile#L12-L17

after `make build` -> there would be a bunch of new files in the skills folder (that we don't want git to track, it's already there in commons)